src/pages/LandingPage/components/EventList/components/EventCard/components/CardStatusLabel/CardStatusLabel.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Container, Label, HelpText } from './styled';
+import { EVENT_STATUS_OPTIONS } from 'src/utils/constants'
+
+type StatusLabelProps = {
+  eventStatus: string;
+};
+
+function CardStatusLabel({ eventStatus }: StatusLabelProps) {
+  if (eventStatus === EVENT_STATUS_OPTIONS.Draft)
+    return (
+      <Container>
+        <Label>
+          Draft<HelpText>This event is only visible to you</HelpText>
+        </Label>
+      </Container>
+    );
+  else {
+    return null;
+  }
```

Comment:
Early returns are preferrable when possible. They're a bit more readable and "safe" as you garantuee to have handled all "edge-cases" before the "normal" case. There's no "edge-case" here, but you can use early return on the less interesting case to make the component a bit more readble, e.g.:
```tsx
if (eventStatus !== EVENT_STATUS_OPTIONS.Draft) {
  return null;
}

return (
  <Container>
    <Label>
      Draft<HelpText>This event is only visible to you</HelpText>
    </Label>
  </Container>
);
```

On a side-note, I also just want to comment that including curly brackets on if-statement is preferrable in my opinion. There's no right answer here, but always including them has the following two advantages:
1. You visually define the end of a branch, so people can move their eyes easier when they want to ignore a case - instead of first having to figure out that it is a one-line branch; and
2. If you change your mind and need to add additional statements to a branch then you won't need to add the curly brackets later - which can be a bit more annoying to do

Reply:
Early returns make a lot of sense.

I agree with your points on curly brackets on if-statements.